.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: CI
+
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    # Run the workflow every Monday at 6 am CST
+    - cron: '0 11 * * MON'
```

Comment:
Curious, this would run the workflow which lints and tests the code:
1. When a push happens to master
2. When a pull request happens to any branch; and
3. Run every Monday.

Is my understanding correct? Instead, should we say "all PRs to `master` branch" should run this workflow of linting and testing? And use a `push` workflow for deployments to say Netlify/elsewhere?

Reply:
Yep, you understood correctly with the 3 events.

The `schedule` (3rd item), which does a regular health check, is likely unnecessary for this repo. At work, my team and I made a separate, scheduled workflow that would run tasks that take more time than tests do. By separating it, we could keep our PR checks short in time.

I'm not quite sure about the `push` workflow that you mentioned. Currently, this app isn't deployed anywhere, I think. Do you think it'd be good to deploy it somewhere?

Reply:
Alternatively, I wonder if we can write a GitHub Actions workflow that can fetch contributors and change the current Times issue to include the list for us. That'd be a good hackathon project.

Reply:
> Do you think it'd be good to deploy it somewhere?

Maybe not @ijlee2 because it is only us using it.